### ISSUE TITLE
'The Item does not exist' error when renaming an item.

### ISSUE THREAD

**Issue by TeusV** [microsoft/AL#5232]
Occurs when the item is used on a sales line, combined with a quantity (and maybe a reservation).

![image](https://user-images.githubusercontent.com/36189834/63688454-c069db80-c807-11e9-85d7-e13d01191a39.png)

Possible solutions:
- In table "Sales Line", procedure RenameNo(), do not use RunTrigger of modifyall() (this is also default behavior for fields with a proper table relation, when a record in the related table is being renamed).
- In codeunit "Sales Line-Reserve", procedure TestSalesLineModification(), replace
    ``` al
      if (NewSalesLine."Bin Code" <> OldSalesLine."Bin Code") and
        (not ReservMgt.CalcIsAvailTrackedQtyInBin(
    ```
    with
    ``` al
      if NewSalesLine."Bin Code" <> OldSalesLine."Bin Code" then
        if not ReservMgt.CalcIsAvailTrackedQtyInBin(
    ```

**Comment by TeusV** [comment]
Replicated this in 14.0.35569 (14.03)

**Comment by atoader** [comment]
Let's get the issue to a team who will fix and backport it, as this repo is only for issues related to the latest developer preview environment for Dynamics 365 Business Central. We suggest that you open a support case or file a bug in Collaborate, to ensure that all Business Central users benefit from your catch as soon as possible.
 
To open a support case, you can:
 
- Open a support request to CSS through the PartnerSource portal 
- Contact your Service Account Manager (SAM) in your local subsidiary to understand what is included in your contract as of support incident and PAH (Partner Advisory Hours). Your SAM might also direct you "step by step" how to open a support request or how to get credentials, if this is the first time for you or your company
 
If you file the bug in Collaborate, remember to include steps to reproduce the issue, and the Business Central build number and country version you're using.

**Comment by TeusV** [comment]
1. I see this answer everywhere and I have no clue how to do this.
2. This **is** an issue related to the latest developer preview environment for Dynamics 365 Business Central. It's still in there, I just checked.
  
From my point of view, I'm just trying to help you improve your product, for free, and I spent time enough already looking where to provide this feedback, reading the guide, creating the issue, and reproducing it on the latest version (August 26 on 14.0.35569 and now on 15.1.37881.38431).
